### ISSUE TITLE
Could not pass event EntityExplodeEvent to Factions v2.8.7Beta

### ISSUE THREAD

**Issue by SmallSansSerif** [MassiveCraft/Factions#1044]
Getting this error on latest builds of Spigot/PaperSpigot. It looks like Spigot added some Ender Dragon commits recently.

http://pastebin.com/9GB0MzRy

**Comment by LHammonds** [comment]
I'm getting the same error message. I came here to report it but saw this one instead.  I compiled my version of Spigot on 2016-03-07 @ 8pm CST (git-Spigot-3104eb1-0705465).  I re-compiled a new version just now (24-hrs since last build) and only 1 version behind the most current.Spigot 1.9 build.  The error also occurs in the latest Spigot build (git-Spigot-3104eb1-5d90f97)

**Comment by Goldentoenail** [comment]
My thanks go out to LHammonds.

I had the same problem and I did not see a solution until I found this.
It would seem that some of the early builds of Spigot 1.9 had a defective world generator.
Creating replacement worlds fixed the problem.

I am using the latest release which is from this morning -
This server is running CraftBukkit version git-Spigot-104c8c8-e43b278 (MC: 1.9) (Implementing API version 1.9-R0.1-SNAPSHOT)
